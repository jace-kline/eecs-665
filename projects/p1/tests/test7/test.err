FATAL [1,21]: Illegal character \
FATAL [1,90]: Empty character literal
FATAL [1,139]: Illegal character ?
FATAL [1,163]: Illegal character \
flex scanner jammed
