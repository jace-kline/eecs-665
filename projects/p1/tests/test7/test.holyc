'\' "This is a test \' to escape a single quote in both a char literal and string literal"
'' "Can I do a single quote followed by another? Another random escape \a
"How about \ this?"