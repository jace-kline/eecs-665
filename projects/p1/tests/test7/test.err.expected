FATAL [1,1]: Bad escape sequence in char literal
FATAL [2,4]: Unterminated string literal  with bad escape sequence ignored
FATAL [3,1]: String literal with bad escape sequence ignored